server/models/Profile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const profileSchema = new Schema({
+    first_name: { type: String, required: true },
+    last_name: { type: String, required: true },
+    description: { type: String, required: true },
+    location: { type: String, required: true },
+    availability: [{
+        from: Date,
+        to: Date
+    }],
+    gender: { type: String, required: true },
+    profile_picture: { type: String, required: false },
+    phone_number: [{ type: Number, required: true }],
+    payment_methods: [{
+        card_type: String,
+        card_number: String, //encrypt later with bcrypt, do not store without encryption
+        card_expiration_date: String, //encrypt later with bcrypt, do not store without encryption
+        card_security_pin: Number, //encrypt later with bcrypt, do not store without encryption
+    }],
```

Comment:
So, we'd actually be utilizing a service like [Stripe](https://www.stripe.com) for all our payment needs and I would feel better storing credit card details on a customer profile instead of here. 

Reason is, financial information is highly sensitive and Stripe has very secure systems so I would much rather utilize that. We could potentially store the customer id from Stripe on the profile and perhaps the last 4 digits on their card also provided from Stripe's customer. 

Let's not focus too much on this and keep it simple and just store the customerId for the time being. Good job on planning for the future.

Reply:
I deleted sample.env because I thought once I made my own .env you should just replace it but of course it makes sense to  keep it for future maintainability and other devs.